wagtailio/areweheadlessyet/blocks.py

```diff
@@ -14,8 +15,13 @@ class LinkBlock(StructBlock):
     link_text = CharBlock(required=True)
 
 
+class _RichTextBlock(RichTextBlock):
```

Comment:
nitpick: 
```suggestion
class RichTextBlockWithExpandedApiRepresentation(RichTextBlock):
```

better to be a bit more verbose for the future.

Reply:
I was exactly going to do something like that but I was hesitating. Thanks, I won't hesitate to be verbose in the future.

Reply:
Thanks! Would you mind approving the PR if it's fine for you please?

Reply:
absolutely. nice one, @Tijani-Dia